qutebrowser/utils/qtutils.py

```diff
@@ -33,7 +34,6 @@
 if TYPE_CHECKING:
     from qutebrowser.qt.webkit import QWebHistory
     from qutebrowser.qt.webenginecore import QWebEngineHistory
-    from typing_extensions import TypeGuard  # added in Python 3.10
```

Comment:
That slipped through my review of imports removed by `ruff`, but it is indeed unused since 5cfab728b29610dd1ab43591b4ba66919621b8a2.